components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtUtil.java

```diff
@@ -1225,4 +1228,31 @@ public static boolean isTrustedAppConsentRequired() {
 
         return Boolean.parseBoolean(IdentityUtil.getProperty(TRUSTED_APP_CONSENT_REQUIRED_PROPERTY));
     }
+
+    /**
+     * check the given tenant is a sub org
+     *
+     * @param tenantDomain
+     * @return true if the tenant is a sub org
+     * @throws OrganizationManagementException
+     */
+    public static boolean isSubOrg(String tenantDomain)
+            throws OrganizationManagementException {
+
+        return OrganizationManagementUtil.isOrganization(tenantDomain);
+    }
+
+    /**
+     * Get the parent organization id of the given organization.
+     *
+     * @param tenantDomain
+     * @return parent organization id
+     * @throws OrganizationManagementServerException
+     */
+    public static String getParentOrgId(String tenantDomain)
```

Comment:
Change the method name and comments as getPrimaryOrgId / getRootOrgId.
Parent org means the immediate org in up, but primary means the root org the tree.